test/controllers/branding_group_controller_test.rb

```diff
@@ -72,8 +82,13 @@ class BrandingGroupControllerTest < ActionDispatch::IntegrationTest
     @collection.reload
     assert @collection.can_edit?(@user)
     assert @collection.can_edit?(new_user)
+    assert_equal 3, @collection.study_list.size
+    assert_equal accessions.sort, @collection.study_list.sort
+    accessions = [studies.first.accession]
+    # test sanitizer logic by using blank space and extra commas
     curator_params = {
-      curator_emails: "#{@user.email}",
+      curator_emails: " #{@user.email}  , ,",
+      study_accessions: "      #{studies.first.accession},,, ",
```

Comment:
You know a comment like line 88 is useful when a reviewer overlooks it at first, and almost requests exactly what you wrote. :)